### ISSUE TITLE
Android & IOS Client Versions needs bumping

### ISSUE THREAD

**Issue by CosmKiwi** [tombulled/innertube#69]
In config.py the client_version for Android and IOS can be bumped to client_version="18.11.34" to avoid HTTP 400 error.

Existing versions:

client_name="ANDROID",
client_version="17.13.3",

client_name="IOS",
client_version="17.14.2",

**Comment by tombulled** [comment]
Hi @CosmKiwi, thanks for opening this issue, I'll look into this now

**Comment by tombulled** [comment]
I've successfully been able to replicate the 400 Bad Request issue. Looks like the below versions are the current latest:
* IOS: 19.16.3 ([released Apr 21, 2024](https://apps.apple.com/us/app/youtube-watch-listen-stream/id544007664))
* ANDROID: 19.17.34 ([released Apr 26, 2024](https://play.google.com/store/apps/details?id=com.google.android.youtube))

I've tested both locally and confirmed that bumping the client versions to the latest versions available solves the issue.

I'll get a PR in for these changes now and will push an updated version of `innertube` with these changes. Many thanks for bringing this issue to my attention :slightly_smiling_face:
